test/opts/opts.spec.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+// as of v6.0.0 we no longer use `mocha.opts` by default, which means
+// we must provide `--opts` on command line to run this test file
+describe('mocha.opts', function() {
+  it('should not duplicate mocha.opts args in process.argv', function() {
+    var processArgv = process.argv.join('');
+    var mochaOpts = fs
```

Comment:
Uh, couldn't you have left this in "regressions.spec.js" and passed `--opts` manually via `process.argv`? For that matter (unless I'm misunderstanding this test), shouldn't we be able to manually set `argv`, invoke `getOptions`, and `expect` as is?

Reply:
No, because the test relies on `--opts` being set when both the test AND fixture are run.  We don't use it on the test any longer (by default), so we need to run it special-like.